src/pages/contact.astro

```diff
@@ -0,0 +1,220 @@
+      }
+      .contact-form button:hover {
+        background-color: #0056b3;
+        transform: scale(1.05);
+      }
+      .contact-form button:active {
+        background-color: #003f7f;
+        transform: scale(0.95);
+      }
+      .contact-form .feedback {
+        display: none;
+        margin-top: 15px;
+        padding: 10px;
+        background-color: #e7f7e7;
+        border: 1px solid #b0d4b0;
+        color: #3a5d3a;
+        border-radius: 5px;
+      }
+      .contact-form .feedback.show {
+        display: block;
+      }
+      @media (min-width: 768px) {
+        .contact-container {
+          flex-direction: row;
+          justify-content: space-between;
+        }
+        .contact-info, .contact-form {
+          flex: 1;
+          margin: 0 20px;
+        }
+        .contact-form button {
+          width: auto;
+        }
+      }
+      @keyframes fadeIn {
+        from {
+          opacity: 0;
+        }
+        to {
+          opacity: 1;
+        }
+      }
+      @keyframes slideIn {
+        from {
+          transform: translateY(20px);
+          opacity: 0;
+        }
+        to {
+          transform: translateY(0);
+          opacity: 1;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <Header />
+    <main>
+      <div class="contact-container">
+        <div class="contact-info">
+          <h2>Contact Us</h2>
+          <div class="info-item">
+            <img src="/images/email-icon.png" alt="Email Icon">
+            <span>Email: email@email.com</span>
+          </div>
+          <div class="info-item">
+            <img src="/images/phone-icon.png" alt="Phone Icon">
+            <span>Phone: 407-555-5555</span>
+          </div>
+          <div class="info-item">
+            <img src="/images/location-icon.png" alt="Location Icon">
+            <span>Address: 123 Mailing Address, Orlando, FL 32825</span>
+          </div>
+        </div>
+        <div class="contact-form">
+          <h2>Questions or Concerns? Fill out the form below.</h2>
+          <form id="contactForm">
+            <input type="text" name="name" placeholder="Your Name*" required>
+            <input type="tel" name="phone" placeholder="Phone Number*" required>
+            <input type="email" name="email" placeholder="Email*" required>
+            <textarea name="message" placeholder="Enter Message Here*" required></textarea>
+            <button type="submit">Submit</button>
+            <div class="feedback" id="feedback">Thank you for your message! We will get back to you soon.</div>
+          </form>
+        </div>
+      </div>
+    </main>
+    <Footer />
+    <script>
+      document.getElementById('contactForm').addEventListener('submit', function(event) {
+        event.preventDefault();
+        const form = event.target;
+        const feedback = document.getElementById('feedback');
+        
+        // Add validation for empty fields
+        const name = form.name.value.trim();
```

Comment:
form.name is not working because `name` is a reserved property of HTMLFormElement. For this you either need to change the name input's name or in the JS code you can wrap it, like so:
 `const name = (form.name as HTMLInputElement).value.trim();`